management_api_app/.env.sample

```diff
@@ -17,7 +17,9 @@ TRE_ID=mytre-dev-3142                           # The Azure TRE instance name -
 # State store configuration
 # -------------------------
 STATE_STORE_ENDPOINT=https://localhost:8081     # The Cosmos DB endpoint - keep localhost if using an emulator. Otherwise https://<your_cosmos_db>.documents.azure.com:443/
-STATE_STORE_KEY=__CHANGE_ME__                   # The Cosmos DB access key
+COSMOSDB_ACCOUNT_NAME=__CHANGE_ME__            # The Cosmos DB account name
```

Comment:
Does this mean we are now requiring an Azure based cosmos? so emulator won't work anymore?

Reply:
Good question, the way that the key retrieval works it uses the RG name and cosmos db name to get the keys. I'm not sure same can be queried against a local instance of cosmosdb. Should we add to the local env template the store_key, and if it exists, it uses it instead of the getting it from credentials?

Reply:
@TessFerrandez fixed now